action.yml

```diff
@@ -83,7 +83,7 @@ runs:
       if: ${{ github.event_name == 'workflow_dispatch' }}
       run: |
         npm version --no-git-tag-version ${{ inputs.semver }}
-        echo "::set-output name=PACKAGE_VERSION::$(echo $(node -p "require('./package.json').version"))"
+        echo "PACKAGE_VERSION=$(echo $(node -p "require('./package.json').version"))" >> $GITHUB_OUTPUT
```

Comment:
can you check if this command still works properly? I realize it's very similar to the previous one, but the nested double quotes are a little suspicious

Reply:
Running the shell command in this same repository gets "PACKAGE_VERSION=3.2.4", which is correct